views/index.ejs

```diff
@@ -1,3 +1,42 @@
 <%- include('partials/head') %>
-</body>
-</html>
\ No newline at end of file
+<main>
+  <section class="introduction-text">
+    <h1> Measuring company impact simplified </h1>
+    <p> Gain insights into your company's environmental and social impact with our innovative tool.
+      Instead of relying solely on theoretical knowledge, observe real-time calculations of handprint and footprint.
+      Understanding how your actions affect the environment and society is crucial for responsible business practices.
+    </p>
+  </section>
+  <section class="signup-form entrance">
+    <form action="POST">
+      <fieldset>
+        <legend>Register</legend>
+        <label> <span>Company Name</span>
+          <input type="text" placeholder="Company Name" required>
+        </label>
+        <label> <span>Email Address</span>
+          <input type="email" placeholder="Email Address" required>
+        </label>
+        <label> <span>Password</span>
+          <input type="password" placeholder="Password" id="password" required>
+        </label>
+      </fieldset>
+      <div class="password-strength-indicator">
+        <p>Password must contain</p>
+        <ul class="requirement-list">
+          <li> <i class="fa-solid fa-circle" alt="Requirement"></i> <span>At least 8 characters length</span> </li>
```

Comment:
Probeer af te stappen van Font Awesome; voor iconen gebruik je beter SVG's.

Reply:
Heb dit ook weggehaald, het is onnodig nu aangezien je een pop-up krijgt met de melding waarin staat wat er in je wachtwoord moet komen!